### ISSUE TITLE
[BUG] Status chart: SOnnet and Opus lines are indistinguishable colors on dark themes

### ISSUE THREAD

**Issue by dusk** [anthropics/claude-code#36479]
### Preflight Checklist

- [x] I have searched [existing issues](https://github.com/anthropics/claude-code/issues?q=is%3Aissue%20state%3Aopen%20label%3Abug) and this hasn't been reported yet
- [x] This is a single bug report (please file separate reports for different bugs)
- [x] I am using the latest version of Claude Code

### What's Wrong?

the lines are the same color.. I have no idea what i'm looking at in stats models and their usage.

### What Should Happen?

they should be different colors 

<img width="641" height="703" alt="Image" src="https://github.com/user-attachments/assets/a32f4001-3479-416c-9fd4-70ecd19f660d" />

### Error Messages/Logs

```shell

```

### Steps to Reproduce

/stats go to models on a dark theme

### Claude Model

Sonnet (default)

### Is this a regression?

I don't know

### Claude Code Version

2.1.80

### Platform

Anthropic API

### Operating System

macOS

### Terminal/Shell

Other

### Additional Information

im using ghostly
